src/app/Dice.elm

```diff
@@ -0,0 +1,19 @@
+module Dice exposing (emptyCup, Cup, cupFromIntList)
```

Comment:
`Dice.Cup` reads more descriptively to me, but we can change it if you like.

I like that name more because I don't have to know anything about the app to understand it straight up. That's my ideal when writing code: that it reads well, and that the reader doesn't have to look up other references to understand as much as possible about the code.

`Dice.Cup` and `Dice.emptyCup` provide this really well to me.

A cup is just a container, to my mind. The important thing here is that it's full of dice.